kagaba-etienne/my-brand
# Ch-node: adding node js backend

#### What does this PR do?

- adds node js backend to the project
- transform HTML pages into views
- email a subscriber immediately after subscribing
- able to respond query by email on the admin panel

#### Description of Task to be completed

- add comment
- delete blog & project: admin is able to perform this action on both blogs and projects
- update blog & project: admin is able to perform this action on both blogs and projects
- publish blog & project: admin is able to perform this action on both blogs and projects
#### How should this be manually tested?

- try subscribing to my newsletter and check your Gmail inbox
- write to me a query on the contact page of the client side and wait for a response

#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
I had done this implementation before but using json-server, which brought some errors when deployed, and had to switch to node js backend.
#### Screenshots (if appropriate)
